src/tedpy/ted6000.py

```diff
@@ -48,11 +44,10 @@ async def update(self) -> None:
 
         """Fetch MTU/Spyder data results"""
         await asyncio.gather(
-            self._update_endpoint("endpoint_dashboard_results", ENDPOINT_URL_DASHBOARD),
             self._update_endpoint("endpoint_mtu_results", ENDPOINT_URL_MTU),
             self._update_endpoint("endpoint_spyder_results", ENDPOINT_URL_SPYDER),
-            *map(lambda mtu: self._update_endpoint(
-                "endpoint_mtu%d_dashboard_results" % mtu.position, 
+            self._update_endpoint("endpoint_dashboard_results", ENDPOINT_URL_DASHBOARD, "0"),
```

Comment:
It'll be a little bit cleaner if the "0" is an integer. That way we can do `self.endpoint_dashboard_reuslts[0]` for retrieval and not need to cast `mtu.position`.

Reply:
It makes the function a bit more reusable if the field is a string but at the cost of some casting in this instance. I'll swap it back, no problem.

Reply:
How about using Any so the caller can decide? 13107f2

Reply:
I like it!